ctapipe/image/pixel_likelihood.py

```diff
@@ -131,7 +135,7 @@ def neg_log_likelihood_numeric(
 
     likelihood = epsilon
 
-    ns = np.arange(*poisson(np.max(prediction)).ppf(confidence))
+    ns = np.arange(poisson(np.max(prediction)).ppf(confidence) + 1)
```

Comment:
What is this change doing? It is not obviously related to the typos in the expressions you are fixing elsewhere.

Reply:
I removed the `*` and changed the `confidence` argument to only give the upper end of the confidence range (Will change docstring btw.). That is because `ns`  needs to always start at 0, and if `np.max(prediction)` is too large and a lower bound is set, that is not the case. the `+1` is just an extra safety margin.